force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -94,7 +127,13 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the input is "Empty", "Null", or "Contains Text".
      */
     public static String checkString(String a) {
-        return null; // Replace null with the variable you used to store the result
+        if(a == null){
+            return 'Null';
+        } else if(a != ''){
+            return 'Contains Text';
+        } else {
+            return 'Empty';
+        }
```

Comment:
Great work on this question. For the future, you can look at String.isEmpty() and String.isBlank() methods, which will accomplish this for us in a real situation.